### ISSUE TITLE
couldn't replicate the results mentioned in your paper

### ISSUE THREAD

**Issue by userid623** [x-28/MAFH#2]
Could you provide following files about the data splits? 

> dataset.nus_wide
> dataset.mirflckr25k

I attempted an experimental configuration with a training set of 10,000 and a test set of 2,000 but couldn't replicate the results mentioned in your paper. Below is my result of 16-bit hash codes on mirflickr:

> MAP of Image to Text: 0.7185, MAP of Text to Image: 0.7088
